challenges/KICKin-Rollin-Challenge.tex

```diff
@@ -0,0 +1,33 @@
+% !TeX root = ../SPL-Challenges.tex
+% !TeX spellcheck = en_US
+\section{KICKin\textquotesingle \& Rollin\textquotesingle Challenge}
+
+The objective of this challenge is for a robot to successfully kick a rolling ball from a ramp into a designated goal area. 
+Each team participating in this challenge can record up to three attempts throughout the competition.
```

Comment:
Right, considering the Open Challenge would take time in the schedule and if we add the other challenge it would also need time in the schedule. I was thinking this would help to reduce pressure on the schedule. 
I understand this would also give less chances for others to see the challenge, but it would give chance to teams to perform once they are ready.

Reply:
Scheduling decisions should not be part of the rules. This is a decision for the OC and TC together as you get closer to the competition. I recommend taking out any text that implies a particular scheduling strategy.